### PR TITLE
update ocean to 2.4.0

### PR DESCRIPTION
**Pull request by mihaisc**

### CODE REVIEW

**Review by EnzoVezzaro — APPROVED** [review]

File: src/@utils/provider.ts [hunk 1]
```diff
@@ -87,7 +88,13 @@ export async function getFileUrlInfo(
   providerUrl: string
 ): Promise<FileInfo[]> {
   try {
-    const response = await ProviderInstance.checkFileUrl(url, providerUrl)
+    const fileUrl: UrlFile = {
```
Comment (kremalicious) [inline src/@utils/provider.ts:91]: the method name `getFileUrlInfo()` does not fit anymore after this change so should be renamed to e.g. `getFileInfo()`

Comment (kremalicious) [reply src/@utils/provider.ts:91]: Forgot that comment, had still the changes from #1732 in mind. So are we going to leave the change to accept an object to cover all the other file types out of this PR?

Comment (kremalicious) [reply src/@utils/provider.ts:91]: alright, should be possible to defer the rest to #1732 as everything works with this PR and we need ocean.js v2.4.0 live, so good to go here

**Review by kremalicious — APPROVED** [review]

File: src/@utils/provider.ts (hunk 1 above)
Comment (mihaisc) [reply src/@utils/provider.ts:91]: This is intended, this function `getFileUrlInfo` just returns file url info.

**Pull request merged**